src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -21,16 +21,27 @@
     UniqueTagList getTags();
 
     /**
-     * Returns true if the values inside this object is same as those of the other
+     * Returns true if this person is the same as that represented in another
+     * object. Two people are the same if their identity fields (name and
+     * telephone) are the same.
```

Comment:
Perhaps this can be more concise. The first statement probably isn't necessary since it conveys the same information as the second statement without adding anything new.

Maybe something like:

`Returns true if both persons have the same personal identity field (name and telephone).`